cli/tsc.rs

```diff
@@ -1751,6 +1757,7 @@ mod tests {
     assert!(
       parse_ts_reference(r#"/ <reference unknown="unknown" />"#).is_none()
     );
+    assert!(parse_ts_reference(r#"/ <asset path="./styles.css" />"#).is_none());
```

Comment:
asset? I haven't heard of this triple slash reference.

Reply:
It's not a valid reference - as reported in #7205 - just making sure it's not caught